x/ecocredit/server/genesis.go

```diff
@@ -264,28 +224,30 @@ func (s serverImpl) ExportGenesis(ctx types.Context, cdc codec.Codec) (json.RawM
 		index++
 	}
 
-	gs := &ecocredit.GenesisState{
-		Params:      params,
-		ClassInfo:   classInfo,
-		BatchInfo:   batchInfo,
-		Sequences:   sequences,
-		Balances:    balances,
-		Supplies:    supplies,
-		ProjectInfo: projectInfo,
-	}
-
-	gs.ProjectSeqNum = s.projectInfoSeq.CurVal(ctx)
+	/*
```

Comment:
a lot of this isn't relevant anymore, so just commenting it out for now and leaving it for #995 976

Reply:
As mentioned above, might make more sense to focus on getting #977 merged first.